src/App.js

```diff
@@ -2,54 +2,49 @@ import './App.css';
 import React from 'react';
 
 import { createApi } from 'unsplash-js';
+const accessKey = "hHYInPsQTpB1kZfbJhhnoq0y1CyB8h6TzStfcD_rf44";
+const endpoint = "https://api.unsplash.com/photos";
 
-const unsplash = createApi({
-  accessKey: "hHYInPsQTpB1kZfbJhhnoq0y1CyB8h6TzStfcD_rf44",
-});
-
-// function App() {
-
-//   const gridRef = React.useRef();
-//   const [rowHeight, setRowHeight] = React.useState();
-//   const [rowGap, setRowGap] = React.useState();
-
-//   React.useEffect(() => {
-//     setRowHeight(parseInt(window.getComputedStyle(gridRef.current).getPropertyValue('grid-auto-rows')));
-//     setRowGap(parseInt(window.getComputedStyle(gridRef.current).getPropertyValue('row-gap')));
-//   });
-
-//   function RenderSquare() {
-//     let w = 250;
-//     let h = Math.floor(Math.random() * 4) * 100 + 200;
-//     let c = Math.floor(Math.random() * 4) + 1;
-//     const rowSpan = Math.ceil((h + rowGap) / (rowHeight + rowGap));
-
-//     return (<div className={`test test${c}`} style={{ width: `${w}px`, height: `${h}px`, gridRowEnd: `span ${rowSpan}` }}>
-//     </div>);
-//   }
-
-//   function Grid(props) {
-//     return (<div className="grid" ref={gridRef}>
-//       {props.children}
-//     </div>);
-//   }
 
 function App() {
-
   const gridRef = React.useRef();
-  const [gridHeight, setGridHeight] = React.useState(2200);
+  const [gridHeight, setGridHeight] = React.useState(5000);
   const [rowGap, setRowGap] = React.useState(10);
+  const [photosArray, setPhotoArray] = React.useState([]);
+  const [columnLeft, setColumnLeft] = React.useState([]);
+  const [columnMid, setColumnMid] = React.useState([]);
+  const [columnRight, setColumnRight] = React.useState([]);
 
   React.useEffect(() => {
-    setGridHeight(parseInt(window.getComputedStyle(gridRef.current).getPropertyValue('height')));
+    //setGridHeight(parseInt(window.getComputedStyle(gridRef.current).getPropertyValue('height')));
+    fetchPhotos();
   }, []);
 
-  function RenderSquare({ h }) {
-    let w = 250;
-    let c = Math.floor(Math.random() * 4) + 1;
-    // const rowSpan = Math.ceil((h + rowGap) / (rowHeight + rowGap));
-    // const finalHeight = Math.ceil((h + rowGap) / (gridHeight + rowGap));
-    return (<div className={`test test${c}`} style={{ width: `${w}px`, height: `${h}px` }}>
+  React.useEffect(() => {
+    assignColumn();
+  }, [photosArray]);
+
+  async function fetchPhotos() {
+    const fetchedPhotos = await fetch(`${endpoint}?page=1&per_page=30`, {
+      headers: {
+        Authorization: `Client-ID ${accessKey}`,
+      }
+    });
+    const result = await fetchedPhotos.json();
+    setPhotoArray(result);
+  }
+
+  // function RenderSquare({ h }) {
+  //   let w = 416;
+  //   let c = Math.floor(Math.random() * 4) + 1;
+  //   return (<div className={`test test${c}`} style={{ width: `${w}px`, height: `${h}px` }}>
+  //   </div>);
+  // }
+
+  function RenderImage({ url, height }) {
```

Comment:
Generally you don't need to add "Render" in component names.

```suggestion
  function Image({ url, height }) {
```